Tasks/ServiceFabricDeployV1/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -74,6 +74,7 @@
   "loc.messages.DIFFPKG_ServiceDoesNotExist": "The service {0} of application {1} to be upgraded by using diff package does not exist in the cluster {2}. Copying it to diff package now...",
   "loc.messages.DIFFPKG_ServiceIsNotChanged": "The service {0} of application {1} to be upgraded by using diff package has the same version {2} running in the cluster {3}. Skip upgrading it.",
   "loc.messages.DIFFPKG_TestAppPkgFailed": "Testing application package failed before creating diff package. Skip creating diff package.",
+  "loc.messages.CertNotPresentInLocalStoreWarningMsg": "The certificate with thumbprint {0} is not present in the local certificate store. This can cause errors.",
```

Comment:
Please get error message from Pulkit. it can be something like: 
The certificate with thumbprint {0} is not present in the certificate store. This can cause errors. It can happen if two releases/builds are running simultaneously on same machine and both of them are deploying to cluster(s) using same service fabric service endpoints. If release/build fails, please re-try it or ensure that multiple agents are not running on same machine.